bool mode
float32 x
float32 y
float32 theta
float32 velocity_linear
float32 velocity_angular
bool stop
